builder/azure/arm/template_factory_test.TestKeyVaultDeployment03.approved.json

```diff
@@ -1,67 +1,67 @@
-{
-  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
-  "contentVersion": "1.0.0.0",
-  "parameters": {
-    "keyVaultName": {
-      "type": "string"
-    },
-    "keyVaultSecretValue": {
-      "type": "securestring"
-    },
-    "objectId": {
-      "type": "string"
-    },
-    "tenantId": {
-      "type": "string"
-    }
-  },
-  "resources": [
-    {
-      "apiVersion": "[variables('apiVersion')]",
-      "location": "[variables('location')]",
-      "name": "[parameters('keyVaultName')]",
-      "properties": {
-        "accessPolicies": [
-          {
-            "objectId": "[parameters('objectId')]",
-            "permissions": {
-              "keys": [
-                "all"
-              ],
-              "secrets": [
-                "all"
-              ]
-            },
-            "tenantId": "[parameters('tenantId')]"
-          }
-        ],
-        "enabledForDeployment": "true",
-        "enabledForTemplateDeployment": "true",
-        "sku": {
-          "family": "A",
-          "name": "standard"
-        },
-        "tenantId": "[parameters('tenantId')]"
-      },
-      "resources": [
-        {
-          "apiVersion": "[variables('apiVersion')]",
-          "dependsOn": [
-            "[concat('Microsoft.KeyVault/vaults/', parameters('keyVaultName'))]"
-          ],
-          "name": "[variables('keyVaultSecretName')]",
-          "properties": {
-            "value": "[parameters('keyVaultSecretValue')]"
-          },
-          "type": "secrets"
-        }
-      ],
-      "type": "Microsoft.KeyVault/vaults"
-    }
-  ],
-  "variables": {
-    "apiVersion": "2015-06-01",
-    "keyVaultSecretName": "packerKeyVaultSecret",
-    "location": "[resourceGroup().location]"
-  }
+{

```

Comment:
> The line endings are normalized because \r\n is evil. 😀 This is the reason the diffs are so noisy.

🎉 Add [?w=1](https://github.com/mitchellh/packer/pull/3693/files?w=1) for cleaner diffs. The downside to this trick is you can't comment on the PR. :\